disnake/ext/commands/__init__.py

```diff
@@ -22,5 +22,6 @@
 from .flag_converter import *
 from .flags import *
 from .help import *
+from .interaction_bot_base import *
```

Comment:
this exposes `InteractionBotBase`, which I'm not sure we want. Perhaps `AppCmdIndex` could be moved to `base_core.py` or a new file instead?

Reply:
`base_core.py` seems like the best file for that. https://github.com/DisnakeDev/disnake/pull/1107/commits/a5802385de007644aacf5d5a03a3b2c685ea1b41